javaparser-core/src/main/java/com/github/javaparser/ast/Modifier.java

```diff
@@ -37,7 +37,8 @@
     VOLATILE,
     SYNCHRONIZED,
     NATIVE,
-    STRICTFP;
+    STRICTFP,
+    TRANSITIVE;

```

Comment:
Ah, I see.

I expected this to resemble `javax.lang.model.element.Modifier` which doesn't contain `TRANSITIVE`. I.e. the "module-only" modifiers are kept separately there. There are separate enums for the different module instruction modifiers, e.g. [here](http://download.java.net/java/jdk9/docs/api/java/lang/module/ModuleDescriptor.Requires.Modifier.html) for `Requires`.